src/main/java/dev/aurelium/auramobs/util/ScaleUtil.java

```diff
@@ -0,0 +1,114 @@
+package dev.aurelium.auramobs.util;
+
+import dev.aurelium.auramobs.AuraMobs;
+import org.bukkit.attribute.Attribute;
+import org.bukkit.configuration.ConfigurationSection;
+import org.bukkit.entity.LivingEntity;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class ScaleUtil {
+
+    private static final LinkedList<ScaleEntry> entries = new LinkedList<>();
+    private static Attribute scaleAttribute;
+    private static AuraMobs plugin;
+
+    private static class ScaleEntry {
```

Comment:
I'm not too keen on this being an inner class, I'd move this to a separate file in the entities package instead of util.